inputsource.cpp

```diff
@@ -181,6 +196,67 @@ void InputSource::cleanup()
     }
 }
 
+#if ENABLE_SIGMF
+void InputSource::readMetaData(const char *filename)
```

Comment:
@miek I've stuck to a null terminated string here to stay in line with `InputSource::openFile()`, but I think just passing a reference to a QString is much nicer. Any preference?

Reply:
I'd be fine with that changing to a QString, as it's internal to the class and that class is all Qt-dependent anyway.

Reply:
Changed.